vercel/turbo
# Update `swc_core` to `v0.87.28`

### Description

next.js counterpart: https://github.com/vercel/next.js/pull/60876

### Testing Instructions

See https://github.com/vercel/next.js/pull/60876

- Closes PACK-2106
- Closes PACK-2257
